src/main/java/LiUni/InputRequests.java

```diff
@@ -0,0 +1,81 @@
+package LiUni;
```

Comment:
everything looks good
please change the package name to all lowercase and you can merge after